deployment/cloud_composer_template.tf

```diff
@@ -133,6 +133,7 @@ resource "google_composer_environment" "example_environment" {
       # Note: keep this in sync with .github/requirements.txt
       pypi_packages = {
         apache-airflow-providers-sendgrid = ""
+	apache-airflow-providers-docker   = ""
```

Comment:
Could you remind me to deploy the change once you merged? Thanks!

Reply:
@RissyRan I'll merge now.